app/views/campaign_requests/_request_details.html.erb

```diff
@@ -2,29 +2,153 @@
+      If applying for Campaigns Platform it is in the form of xxxxx.campaign.gov.uk, or otherwise
+      if a microsite, standard format is  in the form of xxxxxx.gov.uk.
+    </p>
 
-    <%= r.input :proposed_url, label: "Proposed URL (in the form of xxxxx.campaign.gov.uk)", required: true, input_html: { class: "input-md-6", :"aria-describedby" => "urlHelpBlock", :"aria-required" => true } %>
+    <%= r.input :site_metadescription,
+                as: :text,
+                label: "Site metadescription (appears in search results)",
+                required: true,
+                input_html: {:class => "input-md-6",
+                             :rows => 6,
+                             :cols => 50,
+                             :"aria-required" => true,
+                             :"aria-describedby" => "seoHelpBlock" } %>
+    <p class="help-block">
+      The meta description is a short, clear sentence how you want the site described as it appears
+      in Google search
+    </p>
 
-    <%= r.input :site_metadescription, as: :text, label: "Site metadescription (appears in search results)", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true, :"aria-describedby" => "seoHelpBlock" } %>
+    <%= r.input :cost_of_campaign, label: "Site build budget / costs (and overall campaign cost, if applicable)", required: true, input_html: { class: "input-md-6" } %>
+    <p class="help-block">
+      If you are applying for the Campaign Platform page, this is free of any build charge
+    </p>
 
-    <%= r.input :cost_of_campaign, label: "Cost of campaign", required: true, input_html: { class: "input-md-6" } %>
+    <%= r.input :ga_contact_email, label: "Contact details for Google Analytics leads (Gmail accounts only)", required: true, input_html: {:class => "input-md-6", :rows => 6, :cols => 50, :"aria-required" => true } %>
+    <p class="help-block">
+      GCS centralise and coordinate access to Google Analytics, although departments will themselves
+      be responsible for analysing their site’s performance.
+    </p>
+    <p class="help-block">
+      If you need to set up a bespoke analytics package independent of centralised/GCS GA for the purposes
+      of your marketing, please ​provide the following with access ​webmaster@cabinetoffice.gov.uk &
+      microsite​s​@cabinetoffice.gov.uk.
+    </p>
 
     <% end %>
 <% end %>
 
 <%= f.input :additional_comments, as: :text, label: "Additional comments", input_html: { class: "input-md-6", rows: 6, cols: 50 } %>
 
+<p class="help-block">
+  Please attach your OASIS plan using this template (<%= link_to 'OASIS template from the GCS website', 'https://gcs.civilservice.gov.uk/guidance/campaigns/strategy-planning-templates/' %>). In particular you need to demonstrate how the digital / website element of the campaign integrates into your overall OASIS plan and we will need to see <u>specific digital KPIs</u> in the evaluation section.
```

Comment:
The wording here seems likely to confuse users.  "Please attach your OASIS plan" suggests to me that there will be a file upload function, but there isn't.  It seems there's some next stage after this form where the user can attach their OASIS template so we should make it clear that they should have their OASIS plan ready to attach later, but not make them think they can attach it to this form.

Reply:
I've changed the text to "Please note that in the next stage you will also need to provide an OASIS plan using this template" and gotten approval for it.

Reply:
:+1: